modules/troubleshooting-dynamic-plug-in.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins-reference.adoc
+
+:_content-type: REFERENCE
+[id="troubleshooting-dynamic-plug-in_{context}"]
+= Troubleshooting your dynamic plug-in
+
+Refer to this list of troubleshooting tips if you run into issues loading your plug-in. 
+
+* You have enabled your plug-in in the console Operator configuration. Run the following command to check to check if you have enabled your plug-in is in the console Operator configuration and your plug-in name is the output.
++
+[source,terminal]
+====
+oc get console.operator.openshift.io cluster -o jsonpath='{.spec.plugins}'
+====
+
+** You can also look at the enabled plug-ins on the status card of the *Overview* page in the *Administrator perspective*. You will need to refresh your browser if the plug-in was recently enabled.
+
+* Your plug-in service is healthy
+** Check to see if your plug-in pod status is running and your containers are ready.
+** Check to see if the service label selector matches the pod and the target port is correct.
+** Curl the `plugin-manifest.json` from the service in a terminal on the console pod or another pod on the cluster.
+
+* Your `ConsolePlugin` resource name (`consolePlugin.name`) matches the plug-in name used in `package.json`.
+
+* Your service name, namespace, port, and path are declared correctly in the `ConsolePlugin` resource
+
+* Your plug-in service uses HTTPS and service serving certificates.
+
+* Check for certificates or connection errors in the console pod logs.
+
+* Ensure the feature flag your plug-in relys on is not disabled.
+
```

Comment:
Nit: If you do update this list to include a verify statement, my preference is to use the same term in all bullets/steps that do the same action. Picky! Picky!